Ac update readme

Acribbs opened the pull request.

sebastian-luna-valero commented:
Thanks @Acribbs 

I was also updating the README here:
https://github.com/CGATOxford/cgat/blob/SLV-update-install/README.rst

Please have a look and feel free to copy what you consider is missing.

Acribbs commented:
@sebastian-luna-valero Thanks, I added the conda installation into the README and kept the pip but made a comment stating the recommended way of installation was conda. I have also updated the links to the documentation.

Acribbs commented:
@sebastian-luna-valero I have updated the links, please could you make a check to make sure I have changed everything please

sebastian-luna-valero commented:
@Acribbs thanks!
